common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -640,6 +640,8 @@
   "function.wynntils.mana.name": "Mana",
   "function.wynntils.manaMax.description": "Maximum possible mana",
   "function.wynntils.manaMax.name": "Mana[max]",
+  "function.wynntils.manaPct.description": "Your current mana as percentage of max",
+  "function.wynntils.manaPct.name": "Mana[%%]",
```

Comment:
why double %%

Reply:
I think it would have been needed in Legacy, where % were a meta character. It is not needed here.

Reply:
That is how health_pct is written as well so I wanted to stay consistent.

Reply:
okay, sure!

Reply:
Sounds like we should fix health_pct then as well...